specs/protocol/isthmus/predeploys.md

```diff
@@ -243,6 +269,40 @@ The following functions are updated to read from the `L1Block` contract by calli
 - `otherBridge()(address)`
 - `OTHER_BRIDGE()(address)`
 
+### GasPriceOracle
+
+In order to maintain accurate offchain fee estimation, the `GasPriceOracle` must be updated to allow users
+to estimate the operator fee. We also add a new boolean `isIsthmus` to help with evaluating the operator fee.
+
+#### Interface
+
+##### `setIsthmus`
+
+This function is meant to be called once on the activation block of the isthmus network upgrade.
```

Comment:
on second thought maybe this isn't necessary -- if we read the scalars from the l1block before isthmus is set, we'll just read from uninitialized storage, which defaults to 0. So the derived operator fee will just be 0.

Reply:
This is useful for the `GasOracle` predeploy that attempts to do L2 fee estimation

Reply:
You can see [here](https://github.com/ethereum-optimism/optimism/blob/1f106f7868e15a80184145322f165fb8db0bc83c/packages/contracts-bedrock/src/L2/GasPriceOracle.sol#L57) how we have handled this in the past. I believe we will need to include a calculation of the operator fee in the `GasPriceOracle` now